app/mailers/candidate_mailer.rb

```diff
@@ -60,17 +60,18 @@ def new_interview(application_choice, interview)
     )
   end
 
-  def interview_updated(application_choice, interview, old_course)
+  def interview_updated(application_choice, interview, previous_course = nil)
     @application_form = application_choice.application_form
     @interview = interview
     @provider_name = interview.provider.name
-    @course_name_and_code = application_choice.current_course_option.course.name_and_code
-    @old_course_name_and_code = old_course.course.name_and_code
-    @course_has_changed = @course_name_and_code != @old_course_name_and_code
+    @current_course_name_and_code = application_choice.current_course_option.course.name_and_code
+    @previous_course_name_and_code = previous_course&.name_and_code
+    @updated_course_name_and_code =  @previous_course_name_and_code.present? && @previous_course_name_and_code != @current_course_name_and_code
```

Comment:
Rather than a boolean this would need to be
`@updated_course_name_and_code = @current_course_name_and_code if @previous_course_name_and_code.present?` to make it nil or the updated name and code

Reply:
ah I see what you mean now 👍 thanks